airbyte-integrations/connectors/source-tvmaze-schedule/source_tvmaze_schedule/spec.yaml

```diff
@@ -0,0 +1,38 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/tvmaze-schedule
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: TVMaze Schedule Spec
+  type: object
+  required:
+    - start_date
+    - domestic_schedule_country_code
+  additionalProperties: true
+  properties:
+    start_date:
+      type: string
+      description: Start date for TV schedule retrieval. May be in the future.
+      order: 0
+      pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+    end_date:
+      type: string
+      description: |
+        End date for TV schedule retrieval. May be in the future. Optional.
+      order: 1
+      pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+    domestic_schedule_country_code:
+      type: string
+      description: Country code for domestic TV schedule retrieval.
+      examples:
+        - US
+        - GB
+    web_schedule_country_code:
+      type: string
+      description: |
+        Country code for web TV schedule retrieval. Leave blank for all
+        countries plus global web channels (e.g. Netflix). Alternatively, set
+        to 'global' for just global web channels.
```

Comment:
The ISO 3166-1 code country. Please add this info.

Can you add the default to `global` and as example?

Reply:
I've added the ISO code, but not convinced by having global as the default, as leaving it blank is also a valid option that provides the most data.